bitnami/odoo/templates/deployment.yaml

```diff
@@ -120,30 +187,14 @@ spec:
             {{- end }}
           ports:
             - name: http
-              containerPort: 8069
+              containerPort: {{ .Values.containerPort }}
           {{- if .Values.livenessProbe.enabled }}
-          livenessProbe:
-            httpGet:
-              path: /
-              port: http
-            initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds }}
-            periodSeconds: {{ .Values.livenessProbe.periodSeconds }}
-            timeoutSeconds: {{ .Values.livenessProbe.timeoutSeconds }}
-            successThreshold: {{ .Values.livenessProbe.successThreshold }}
-            failureThreshold: {{ .Values.livenessProbe.failureThreshold }}
+          livenessProbe: {{- omit .Values.livenessProbe "enabled" | toYaml | nindent 12 }}
```

Comment:
I though we agreed on hardcoding the probe method (`httpGet` in this case) and exposing only the threshold/seconds parameters to have opinionated probes. Keeping the possibility of using `customXXXProbe` for those users interested on customizing them.

Reply:
OK, reverted.